lib/figures/king.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative 'figure'
+
+class King
+  attr_reader :row, :column, :possible_moves, :color, :display
+
+  include Figure
```

Comment:
including is for code reusing, inheritance is for specialisation. 

It's not a critical thing, but I think it would feel better if `Figure` was a class and you inherited from it. maybe if `def initialize` was in the parent class, it would be quite a bit cleaner :thinking:

Reply:
Every figure has a different move pattern, how would I move them to one initialize? With a switch statement?